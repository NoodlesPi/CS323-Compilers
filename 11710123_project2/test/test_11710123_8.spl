struct apple{
    int value;
    char p;
};
int helloworld(){
    struct apple a;
    return a.value;
}

int main(){
    float f = helloworld();
    return helloworld(1);
}