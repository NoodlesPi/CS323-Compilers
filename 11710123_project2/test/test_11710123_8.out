Error type 5 at Line 11: unmatching type
Error type 9 at Line 12: the functionâ€™s arguments mismatch the declared parameters
